### PR TITLE
Bug 1306226 - Increase cell execution timeout

### PR DESCRIPTION
**Pull request by maurodoglio**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/emr-bootstrap-spark/44)

### CODE REVIEW

File: ansible/files/batch.sh [hunk 1]
```diff
@@ -79,7 +79,7 @@ else
     pyspark
     EXIT_CODE=$?
     if [ $EXIT_CODE != 0 ] || [ "`grep  '\"output_type\": \"error\"' $NOTEBOOK_NAME`" ] ;then
-        PYSPARK_DRIVER_PYTHON=jupyter PYSPARK_DRIVER_PYTHON_OPTS="nbconvert --to markdown --stdout ${NOTEBOOK_NAME}" pyspark
+        PYSPARK_DRIVER_PYTHON=jupyter PYSPARK_DRIVER_PYTHON_OPTS="nbconvert --ExecutePreprocessor.timeout=36000 --to markdown --stdout ${NOTEBOOK_NAME}" pyspark
```
Comment (vitillo) [inline ansible/files/batch.sh]: What about disabling this altogether with `-1`?

Comment (maurodoglio) [inline ansible/files/batch.sh]: yeah sounds good

**Comment by vitillo** [comment]
r+

**Comment by vitillo** [comment]
Deployed

**Pull request merged**